packages/landing/src/widgets/landing/landing.tsx

```diff
@@ -56,11 +56,12 @@ export function Landing() {
 
   return (
     <main>
+      <div id="top-anchor" />
       <Wrapper>
```

Comment:
Let's try to set the `id` attribute on `main` or `Wrapper` not adding new elements.